app/lib/rocdev_web/controllers/social_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule RocdevWeb.SocialController do
+  use RocdevWeb, :controller
+
+  def meetup(conn, _params) do
```

Comment:
Why bother with a controller for this if we're just redirecting?  Why not just put the urls in the page directly?

Reply:
They won't for long. I'd like the Meetup page to show the upcoming events. One issue I have with Meetup's app is that discovery is very difficult if you're not authenticated.

I might also aggregate other known Meetups on this page and provide a single RSS.